test/TesterInternal/CodeGeneratorTests_TakeSerializedData.cs

```diff
@@ -20,7 +20,7 @@ public class CodeGeneratorTests_TakeSerializedData : HostedTestClusterEnsureDefa
             await grain.TakeSerializedData(new Dtos.ClassNotReferencingOrleansTypeDto { MyProperty = "Test" });
         }
 
-        [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("CodeGen"), TestCategory("Serialization")]
+        [SkippableFact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("CodeGen"), TestCategory("Serialization")]
```

Comment:
@sergeybykov: I simply made an automerge via github webclient. And after fetching the branch, everything looks fine in my workspace: [Fact] has been replaced by [SkippableFact].

Sorry, I'm just trying to get familiar with the github stuff – maybe I'm missing something.